### PR TITLE
C# SDK change with Routing Properties

### PR DESCRIPTION
**Pull request by ssankar1984**
Routing Properties changes for C# SDK
Swagger Spec https://github.com/Azure/azure-rest-api-specs/pull/1194

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ Y] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ Y] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ Y] Title of the pull request is clear and informative.
- [ Y] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ Y] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [Y ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [Y ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [Y ] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Bump up the version in csproj, AssemblyInfo.cs
Remove all traces of global.json, project.json and *.xproj from your SDK and SDK Test project directories.

**Comment by shahabhijeet** [comment]
@azuresdkci add to whitelist

File: src/SDKs/IotHub/Management.IotHub/Microsoft.Azure.Management.IotHub.csproj [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <Description>Provides management capabilities for Microsoft Azure IotHub.</Description>
```
Comment (shahabhijeet) [inline src/SDKs/IotHub/Management.IotHub/Microsoft.Azure.Management.IotHub.csproj]: @ssankar1984 please remove all commented code from csproj files.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/IotHub/IotHub.Tests/Helpers/IotHubTestUtilities.cs [hunk 2]
```diff
@@ -1,24 +1,36 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
+﻿//
+// Copyright (c) Microsoft.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
```
Comment (shahabhijeet) [inline src/SDKs/IotHub/IotHub.Tests/Helpers/IotHubTestUtilities.cs]: @ssankar1984 why change the header and use the apache license?
We have adopted to use MIT license, please retain the previous license header

File: src/SDKs/IotHub/IotHub.Tests/ScenarioTests/IotHubLifeCycleTests.cs [hunk 3]
```diff
@@ -1,5 +1,19 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
+﻿//
+// Copyright (c) Microsoft.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
```
Comment (shahabhijeet) [inline src/SDKs/IotHub/IotHub.Tests/ScenarioTests/IotHubLifeCycleTests.cs]: @ssankar1984 retain previous license header

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

**Comment by ssankar1984** [comment]
Swagger PR here - https://github.com/Azure/azure-rest-api-specs/pull/1200

**Pull request merged**